src/router/routes.js

```diff
@@ -119,6 +119,15 @@ export default [
 		path: '/get-started',
 		redirect: '/',
 	},
+	{
+		path: '/gfm-beta',
```

Comment:
Is there a general reason for this to be in `ui` vs `cms-page-server`?

Reply:
Yea, it's an account/portfolio type page that is likely to relate aesthetically to mykiva and eventually contain components that also moving into mykiva.